### PR TITLE
Add troubleshooting information.

### PR DESCRIPTION
**Pull request by bgravenorst**
Signed-off-by: Byron Gravenorst <byron.gravenorst@consensys.net>

## Pull Request Description

Add troubleshooting information.

### CODE REVIEW

File: docs/HowTo/Troubleshoot/Troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+* Set the permissions of the directory holding the keystores so that it is writable by Teku.
+* Set [`--validators-keystore-locking-enabled`](../../Reference/CLI/CLI-Syntax.md#validators-keystore-locking-enabled)
+    to `false` to disable the locking functionality.
+
+## Keystore file already in use
+
+If Teku fails to start with a `Keystore file <keystore_file>.lock already in use.` error, this
+could mean the keystore file is already being used by a validator client, or Teku has exited
+unexpectedly and did not removed the lock.
```
Comment (ajsutton) [inline docs/HowTo/Troubleshoot/Troubleshooting.md:35]:
```suggestion
unexpectedly and did not remove the lock.
```

File: docs/HowTo/Troubleshoot/Troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+Teku uses a lockfile mechanism for the keystores to prevent two validator clients using the same
+keystores at the same time, similar to the Lighthouse client.
+
+To resolve this issue, try the one of the following:
+
+* Manually remove the lock files that are created alongside your keystore files, with `.lock`
+    appended to the filename. Take care not to delete your keystores.
```
Comment (ajsutton) [inline docs/HowTo/Troubleshoot/Troubleshooting.md:43]: Should add a note to double check that no other process is using those keys.  If it is you'll likely wind up getting slashed.

**Review by ajsutton — APPROVED** [review]
LGTM.

File: docs/HowTo/Troubleshoot/Troubleshooting.md (hunk 2 above)
Comment (bgravenorst) [reply docs/HowTo/Troubleshoot/Troubleshooting.md:43]: Done.

**Pull request merged**